src/v/config/configuration.cc

```diff
@@ -259,6 +259,21 @@ configuration::configuration()
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       64,
       {.min = 1, .max = 16384})
+  , raft_replica_max_not_flushed_bytes(
+      *this,
+      "raft_replica_max_not_flushed_bytes",
+      "Max not flushed bytes per partition. If configured threshold is reached "
+      "log will automatically be flushed even tho it wasn't explicitly "
+      "requested",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      2_MiB)
```

Comment:
What do you think about lowering this to something like 256KB? From my testing this is a lot more smooth at higher partition counts. Flushing more often also has the important advantage that the whole replay might happen from the batch cache.

Even with a single partition flushing only every 256KB should be at least an order of magnitude less than what `acks=-1` would do (Unless massive messages are at play?).

Reply:
done